web.py

```diff
@@ -40,8 +41,14 @@
 
 def process_email_alerts():
     with app.app_context():
-        #utils.email_notification_alert(logger, subject, msg, recipient)
-        pass
+        data = dasset.account_balances()
+        for balance in data:
+            if balance.symbol == 'NZD':
+                balance_format = dasset.asset_dec_to_str(balance.symbol, balance.available)
+                subject = f"Available {balance.symbol} Balance below the minimum threshold"
```

Comment:
move the subject and msg declaration lines into block within the balance.available check

Reply:
also balance_format, there is no point executing the code until we know we need to

Reply:
move the subject and msg